README.md

```diff
@@ -1,2 +1,341 @@
+
+```python
+PSP_QMONEY_GRANTTYPE = os.environ.get('PSP_QMONEY_GRANTTYPE')
+PSP_QMONEY_USERNAME = os.environ.get('PSP_QMONEY_USERNAME')
+PSP_QMONEY_PASSWORD = os.environ.get('PSP_QMONEY_PASSWORD')
+PSP_QMONEY_AUTH_BEARER_TOKEN= os.environ.get('PSP_QMONEY_AUTH_BEARER_TOKEN')
+
+PSP_QMONEY_AUTH_URL = os.environ.get("PSP_QMONEY_AUTH_URL")
+PSP_QMONEY_URL_PAYMENT = os.environ.get("PSP_QMONEY_URL_INITIATE")
+PSP_QMONEY_URL_PROCESS = os.environ.get("PSP_QMONEY_URL_PROCESS")
+
+PSP_QMONEY_NAME = os.environ.get('PSP_QMONEY_NAME')
+PSP_QMONEY_ACCOUNT = os.environ.get('PSP_QMONEY_ACCOUNT')
+PSP_QMONEY_PIN = os.environ.get('PSP_QMONEY_PIN')
+
+QMONEY_API_NAME =  os.environ.get('QMONEY_API_NAME')
+QMONEY_API_ACCESS_TOKEN = os.environ.get('QMONEY_API_ACCESS_TOKEN')
+QMONEY_API_ACCESS_TOKEN_EXPIRY = os.environ.get('QMONEY_API_ACCESS_TOKEN_EXPIRY')
+```
+This configuration code is necessary to enable Qmoney integration within your application and should be placed in the specified settings file.
+
+## Creating a Payment Service Provider for Qmoney
+To set up a Payment Service Provider for Qmoney, we have a test account for receiving payments and a Qmoney wallet number for OTP delivery via email (configurable by Qmoney). Here are the test account details:
+
+The following can be added in your env file before any migration so when you run migration it automatically adds it to your database with you having to add it through the Django Admin
+- `name`: Qmoney
+- `account`: 14001502
+- `pin`: 1234
+
+The following can be added in an insurees detail of insuree_wallet
+- `insuree_wallet`: 5811724c
+
+ Please keep in mind that these details are meant for testing purposes. If you intend to transition to a production environment, feel free to update them to align with your production settings.
+
+
+
+
+# Qmoney Mutation Example
+
+#### Initiate Transaction Mutation sample
+```
+mutation {
+
+    initiateTransaction(input:{
+    amount:5.00
+    paymentServiceProviderUuid:"5c15ecbd-ef82-40e0-9842-a5a28cedb2b6"
+    insureeUuid:"0EFC5C60-AC2E-4F2C-840D-022271005D5A"
+
+    }),{
+    responseMessage
+    Success
+    uuids
+    clientMutationId
+  }
+  }
+```
+
+#### Process Transaction Mutation sample
+
+```
+mutation {
+
+    processTransaction(input:{
+    uuid:"5ff3e97e-f765-413f-929c-6c9044ccaae6"
+    amount:5.00
+    otp:"364407"
+    paymentServiceProviderUuid:"5c15ecbd-ef82-40e0-9842-a5a28cedb2b6"
+    insureeUuid:"0EFC5C60-AC2E-4F2C-840D-022271005D5A"
+    clientMutationId:""
+    }),{
+
+  responseCode
+  responseMessage
+  clientMutationId
+  Success
+  detail
+  clientMutationId
+
+
+  }
+ 
+ 
+  }
+```
+
+## Second Workflow 
+
+# Accessing Custom Endpoints
+
+The Mobile Payment module exposes two essential endpoints: `Verify_insuree` and `Process_payment`. These endpoints are designed to be utilized by other payment service providers that wish to enable policyholders to make payments through their services, .
```

Comment:
it's not clear how Afrimoney knows that it should call both endpoints. Could you explain that to me please?

Reply:
their is an api documentation for this

Reply:
Ok, could you tell me where we can find that documentation?

Reply:
i will share that with you tomorrow , will add it in the workflow folder